src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid $card-border-color;
+  background: $card-bcg-color;
+  padding: $card-padding;
+  display: flex;
+  flex-direction: column;
+  transition: $transition-effect;
+  cursor: pointer;
```

Comment:
pay attention please to the css organize rules
https://9elements.com/css-rule-order/